### PR TITLE
Dj/update models

### PR DESCRIPTION
**Pull request by drujensen**

### CODE REVIEW

File: getting-started/models/crecto/README.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
+# Crecto
```
Comment (eliasjpr) [inline getting-started/models/crecto/README.md]: This is great!

**Comment by drujensen** [comment]
@eliasjpr I would like to get this structural change merged.  I will enhance each section (including the migrations) over time but this is a good start.

File: getting-started/models/crecto/README.md (hunk 1 above)
Comment (drujensen) [reply getting-started/models/crecto/README.md]: Yeah, I agree.  I just copied the readme for now.

Comment (drujensen) [reply getting-started/models/crecto/README.md]: @fridgerator maybe you can help with this?  I haven't actually ever used cretco.  :-)

**Pull request merged**